tests/test_nodes_no_host.py

```diff
@@ -62,3 +62,47 @@ def test_algorithm():
     assert a.type == "integration"
 
     # Add citation test, once we have citation implemted
+
+
+def test_local_search():
```

Comment:
I would split up the different tests that comprise this test for clarity

Reply:
You mean having a separate test file for the search?
Yeah, that is an idea.